opencensus/common/resource.py

```diff
@@ -14,13 +14,36 @@
 
 
 from copy import copy
+import os
 import re
 
 
+OC_RESOURCE_TYPE = 'OC_RESOURCE_TYPE'
+OC_RESOURCE_LABELS = 'OC_RESOURCE_LABELS'
+
 # Matches anything outside ASCII 32-126 inclusive
 NON_PRINTABLE_ASCII = re.compile(
     r'[^ !"#$%&\'()*+,\-./:;<=>?@\[\\\]^_`{|}~0-9a-zA-Z]')
 
+# Label key/value tokens, may be quoted
+WORD_RES = r'(\'[^\']*\'|"[^"]*"|[^\s,=]+)'
+
+KV_RE = re.compile(r"""
```

Comment:
Do we intend this to be _public_ (consumed directly by the users)? If not, we can add `_` prefix to the name.

Reply:
I've erred on the side of leaving things public since I think it's a losing battle to add a leading underscore to everything that's not in the API, but I think you're right that it's worth hiding module level constants like this since it's less obvious which ones are supposed to be part of the API.

Reply:
I actually don't have strong opinions since I heard about the Python philosophy `We are all consenting adults here.`.

Looking at the current code, we do follow the underscore convention for constants though (at least in the majority of files).

https://github.com/census-instrumentation/opencensus-python/blob/master/opencensus/trace/trace_options.py#L18
https://github.com/census-instrumentation/opencensus-python/blob/master/opencensus/trace/tracestate.py#L18